x/evm/types/denom.go

```diff
@@ -75,6 +75,11 @@ var ChainsCoinInfo = map[string]EvmCoinInfo{
 		DisplayDenom: types.DisplayDenomTestnet,
 		Decimals:     EighteenDecimals,
 	},
+	utils.TestingChainID: {
+		Denom:        types.BaseDenom,
+		DisplayDenom: types.DisplayDenom,
+		Decimals:     EighteenDecimals,
+	},
```

Comment:
as mentioned in another comment, I think having the chain-specific stuff in the public package is bad abstraction - I've removed it on the OS repo